CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Checking deployments in the cluster namespace using labels.
```

Comment:
```suggestion
- Use managed-by label to check node deployments are deleted before cluster namespace.
```

I think we should mention this is using the managed-by label. WDYT?

Reply:
Looks better, Thanks.